src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -187,5 +187,48 @@ public void SetAccessControl_FileStream_FileSecurity_Success()
                 FileSystemAclExtensions.SetAccessControl(fileStream, fileSecurity);
             }
         }
+
```

Comment:
@JeremyKuhne I added one more test. I am not sure what else I should verify. Can you please take a look and tell me what you think?